README.md

```diff
@@ -625,20 +625,30 @@ symbols helping you easily navigate the Rego code in your workspace.
 
 The Regal language server currently supports the following LSP features:
 
-- [x] Diagnostics (linting)
-- [x] Hover (for inline docs on built-in functions)
-- [x] Go to definition (ctrl/cmd + click on a reference to go to definition)
-- [x] Folding ranges (expand/collapse blocks, imports, comments)
-- [x] Document and workspace symbols (navigate to rules, functions, packages)
-- [x] Inlay hints (show names of built-in function arguments next to their values)
-- [x] Formatting
-- [x] Code completions
-- [x] Code actions (quick fixes for linting issues)
+- [x] [Diagnostics](https://github.com/StyraInc/regal/blob/main/docs/language-server.md#diagnostics) (linting)
```

Comment:
All the links here should be updated to point to docs.styra.com later... but since those aren't there yet, I'll do this in the meantime.